android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -106,30 +99,33 @@
 
         setRepeatingGroupNumLimits(widgetArgs);
 
+        JSONObject countFieldObject = Utils.getRepeatingGroupCountObj(widgetArgs);
+
         //get the generated number of groups
-        final String generatedGroupsCount = jsonObject.optString(REPEATING_GROUPS_GENERATED_COUNT);
+        final String generatedGroupsCount = countFieldObject != null ? countFieldObject.optString(VALUE) : "";
```

Comment:
Based on https://github.com/OpenSRP/opensrp-client-native-form/pull/504/files#r512021105. Should `countFieldObject` ever be null?

Reply:
yeah for malformed json form

Reply:
https://github.com/OpenSRP/opensrp-client-native-form/pull/504#discussion_r513202094